api/data_refinery_api/views.py

```diff
@@ -1336,6 +1342,19 @@ def get(self, request, format=None):
 # QN Targets
 ###
 
+class QNTargetsAvailable(APIView):
+    """
+    This is a list of all of the organisms which have available QN Targets
+    """
+    def get(self, request, format=None):
+        
+
+        organism_ids = list(ComputationalResultAnnotation.objects.filter(data__is_qn=True).values_list('data__organism_id', flat=True).order_by('data__organism_id'))
+        organisms = Organism.objects.filter(id__in=organism_ids)
```

Comment:
Seems that we use this in multiple places, is it worthwhile to move it into `Organism`? Similarly to `Experiments.public_objects`